### PR TITLE
Add mappings for ReadyToRun helpers used by llilc.

### PR DESCRIPTION
**Pull request by erozenfeld**
Add READYTORUN_HELPER_ThrowNullRef and READYTORUN_HELPER_ThrowDivZero.

Allow ReadyToRun tests to be run with llilc: don't throw exceptions when llilc is enabled.

### CODE REVIEW

**Comment by erozenfeld** [comment]
@jkotas PTAL

File: tests/src/readytorun/main.cs [hunk 1]
```diff
@@ -310,6 +318,11 @@ static void RunAllTests()
 
     static int Main()
     {
+        LLILCJitEnabled =
```
Comment (jkotas) [inline tests/src/readytorun/main.cs]: Maybe nice to add a comment here - that this should be removed once exceptions work in LLILC

Comment (erozenfeld) [inline tests/src/readytorun/main.cs]: Done.

**Comment by erozenfeld** [comment]
The tests were failing when complus_altjitname wasn't set. Fixed that.

**Comment by erozenfeld** [comment]
@dotnet-bot rebuild this

**Comment by jkotas** [comment]
@dotnet-bot test this please

**Comment by erozenfeld** [comment]
@jkotas Can you please merge this?

**Pull request merged**